<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="windowTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>450</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Window Type</string>
   </property>
   <widget class="QComboBox" name="windowTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rectangular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hamming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hanning</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kaiser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chebyshev</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="stopBandAttenuationUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dB</string>
    </property>
   </widget>
   <widget class="QLabel" name="rippleUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dB</string>
    </property>
   </widget>
   <widget class="QLabel" name="transitionWidthUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transitionWidthText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="transitionWidthLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Transition Width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rippleText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rippleLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ripple:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopBandAttenuationLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Band Attenuation:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stopBandAttenuationText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="alphaLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alphaText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="filterTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter Type</string>
   </property>
   <widget class="QComboBox" name="filterTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Band Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Band Stop</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lowCutoffFreqUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="highCutoffFreqUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="highCutoffFreqLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>High Cutoff Frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="highCutoffFreqText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lowCutoffFreqLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Cutoff Frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lowCutoffFreqText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nyquistFreqLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nyquist frequency:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nyquistFreqText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="indent">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="nyquistFreqUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="filterWidthGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter Width</string>
   </property>
   <widget class="QComboBox" name="filterWidthComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Points</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Time</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="numPointsLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Points:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numPointsText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timespanLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timespan:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timespanText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timespanUnit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>filterTypeComboBox</tabstop>
  <tabstop>lowCutoffFreqText</tabstop>
  <tabstop>highCutoffFreqText</tabstop>
  <tabstop>filterWidthComboBox</tabstop>
  <tabstop>numPointsText</tabstop>
  <tabstop>timespanText</tabstop>
  <tabstop>windowTypeComboBox</tabstop>
  <tabstop>alphaText</tabstop>
  <tabstop>stopBandAttenuationText</tabstop>
  <tabstop>rippleText</tabstop>
  <tabstop>transitionWidthText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
